src/app/screens/ConfirmPayment/index.tsx

```diff
@@ -157,18 +159,22 @@ function ConfirmPayment() {
           </div>
         </Container>
       ) : (
-        <Container maxWidth="sm">
-          {navState.origin && (
-            <PublisherCard
-              title={navState.origin.name}
-              image={navState.origin.icon}
-              url={navState.origin.host}
-            />
-          )}
-          <div className="my-4">
-            <SuccessMessage
-              message={successMessage}
-              onClose={() => window.close()}
+        <Container justifyBetween maxWidth="sm">
+          <ResultCard
+            isSuccess
+            message={
+              !navState.origin
+                ? successMessage
+                : `${invoice.satoshis} (SATS) ${
+                    showFiat ? "(" + fiatAmount + ")" : ""
+                  } ${tCommon("were_sent_to")} ${navState.origin.name}`
```

Comment:
Should we also show the amount sent even when there's no receiver name?

Reply:
Why not, right?  
Wanna switch `"(" + fiatAmount + ")"` to template-literals?

Reply:
Sure!